incubator/istio/README.md

```diff
@@ -18,7 +18,29 @@ This chart bootstraps a [Istio](https://istio.io/) deployment on a [Kubernetes](
 ## Prerequisites
 
 - Kubernetes 1.5+
-- istioctl - See installation steps [here](https://istio.io/docs/tasks/installing-istio.html#installation-steps)
+- istioctl
+
+### istioctl installation steps
+
+1. Go to the [Istio release](https://github.com/istio/istio/releases) page, to download the installation file corresponding to your OS or run 
+   ```bash
+   curl -L https://git.io/getIstio | sh -
+   ```
+   to download and extract the latest release automatically (on MacOS and Ubuntu).
+
+2. Extract the installation file, and change directory to the location where the files were extracted. The following instructions 
+   are relative to this installation directory.
+   The installation directory contains:
+    * yaml installation files for Kubernetes
+    * sample apps
+    * the `istioctl` client binary, needed to inject Envoy as a sidecar proxy, and useful for creating routing rules and policies.
+    * the istio.VERSION configuration file.
+
+2. Add the `istioctl` client to your PATH. For example, run the following commands on a Linux or MacOS system:
+
+   ```bash
+   export PATH=$PWD/bin:$PATH
```

Comment:
the https://git.io/getIstio already covers that but otherwise you could put here
```bash
export PATH=$PWD/istio-0.1.6/bin:$PATH
```
And telling people to replace 0.1.6 by the version they downloaded